utils/resource/usage.go

```diff
@@ -5,9 +5,11 @@ package resource
 
 import (
 	"math"
+	"strconv"
 	"sync"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/shirou/gopsutil/process"
```

Comment:
For different repos we typically separate the imports.
```suggestion
	"github.com/prometheus/client_golang/prometheus"

	"github.com/shirou/gopsutil/process"
```

Reply:
done